packages/rx_bloc_cli/doc/Fastlane.md

```diff
@@ -0,0 +1,82 @@
+
+---
+
+For iOS, grab the latest provisioning profiles for each supported flavor from the [Apple Developer Console][apple_developer_console].
+
+Additionally, create and export a distribution certificate in `.p12` format. It is suggested to use a strong password when exporting the certificate.
+
+Create a file named `distributionCertificatePassword.txt` and paste the password inside.
+
+Create a new strong password and paste it inside a newly created file named `keychainPassword.txt` . 
+This password will be used for a keychain used for installing all the necessary certificates and provisioning profiles for the iOS build.
+
+Optionally, if you want to deploy to Test Flight directly after the app has been built, you'll need to create an [App Store Connect API token][ios_auth_key_creation] and download it. The file name should have the following format: `AuthKey_{KEY_ID}.p8` where the `KEY_ID` is your unique id.
+
+---
+
+For Android, prepare the keystore that will be used for signing of the app bundle or create a new one using the following command:
+
+```
+keytool -genkey -v -keyalg RSA -keysize 2048 -validity 10000 -keystore android.jks -alias {alias_name} -keypass {key_password} -storepass {store_password}
+```
+
+Make sure the keystore file is named `android.jks` and take note of the alias name, key password and store password. 
+For each of them, create a separate file named `keyAlias.txt`, `keyPassword.txt` and `storePassword.txt` and paste the respective contents inside
+
+Optionally, if you want to deploy to Google Play directly after the app has been built, you'll need to setup a service account within the Google Cloud Platform Console page. Assign a new `.json` key to that service account and downlaod it for later use. Copy the generated email address of the service account and invite it as an Admin in the [Google Play Console][android_developer_console]. Additionally, make sure that the [Google Play Developer API][google_play_developer_api] is enabled on your existing project.
+
+_Note: For an in-depth guide on how to setup a service account and assign it a key, please check the [setup section of Uploading to Play Store][android_service_key_creation] provided by Fastlane._
+
+#### Distribution repository:
+
+Create a new Github repository used for storing files and credentials for distribution of your app. 
+Clone the repository and open the directory in the terminal. 
+Using the [rx_bloc_cli][rx_bloc_cli_link] package, execute the following command:
+
+```
+rx_bloc_cli create_distribution .
+```
+
+The `create_distribution` command will populate the directory with the file structure required for things to work.
+Within the `android` and `ios` directories, paste the previously downloaded files and rename them accordingly. 
+The generated `README` file contains details on how to name the files, if additional help is needed.
+
+You may also need to set the `MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD` variable in your terminal. 
```

Comment:
"You may also need to set the" I think this is a requirement ? If so, communicate it in this manner. Reading "you may" it sounds as an optional step

Reply:
I would say it's semantics. I've seen this form of "polite doublespeak (or doubletalk)" used online in articles to soften the strictness of the imperative talk. 

To back that claim up, we have `need` after it (which by itself means `required`). Instead of saying:

```
You may also need to set the `MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD` variable in your terminal. 
```

to express optionality, we could just say:

```
You may also set the `MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD` variable in your terminal. 
```

---

However, since this brought up confusion on your end, I'll change it to something clearer.